walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,170 @@
 
 package com.google.sps;
 
+import java.lang.Math;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 
 public final class FindMeetingQuery {
+  /**
+   * Determine available times for a meeting to take place given a list of Events and a Meeting Request.
+   * @param events the list of events to take into consideration when finding available times
+   * @param request the meeting request with meeting attendees and duration
+   * @return A list of available TimeRanges
+   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // Meeting cannot last longer than 24 hours
+    if(request.getDuration() > TimeRange.WHOLE_DAY.duration()) return Arrays.asList();
```

Comment:
nit: Here and below, can you add a "space" between the `if` and the initial left paren "(" of the condition?